tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -206,13 +209,373 @@ public function test_get_datapoints() {
 				'create-webdatastream',
 				'properties',
 				'property',
+				'report',
 				'webdatastreams',
 				'webdatastreams-batch',
 			),
 			$this->analytics->get_datapoints()
 		);
 	}
 
+	public function test_get_report() {
```

Comment:
As described in **Relevant technical choices**, I've opted to write these higher-level tests which provide better coverage rather than individually testing the protected methods as described in the IB.